src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -125,11 +122,6 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
 
-  <PropertyGroup>
-    <!-- Suppress NuGet warning about the explicit downgrade to System.Collections.Immutable v1.1.37 -->
-    <NoWarn>NU1605</NoWarn>
-  </PropertyGroup>
-
```

Comment:
We don't reference the MS.CodeAnalysis NuGet packages, which means we don't need to reference specific versions of the `System.Collections.Immutable` or `System.Reflection.Metadata` assemblies either.

That in turn means we no longer need to suppress this NuGet warning.